dev-libs/libmcrypt/libmcrypt-2.5.8.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="libmcrypt are intended to be replacements for the \
+old Unix crypt, except that they are under the GPL and \
+support an ever-wider range of algorithms and modes."
+HOMEPAGE="http://mcrypt.sourceforge.net/"
+COPYRIGHT="1998-2002 Nikos Mavroyanopoulos"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/mcrypt/libmcrypt-$portVersion.tar.gz"
+CHECKSUM_SHA256="e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.4.8"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libmcrypt$secondaryArchSuffix = $portVersion
+	lib:libmcrypt$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmcrypt${secondaryArchSuffix}_devel = $portVersion
+	devel:libmcrypt$secondaryArchSuffix = $libVersion
+	"
```

Comment:
Oops, I forgot to spot a missing line in `PROVIDES_devel`. When you run haikuporter with the `--strict-policy` option (or its `-S` alias), it helps you find unadvertised shared library dependencies, as well as unadvertised `cmd:` items in `PROVIDES` and `PROVIDES_devel`. Here we have a `libmcrypt_config` in `bin/` so we need to add a line for it in the `PROVIDES_devel`:
```diff
 PROVIDES_devel="
 	libmcrypt${secondaryArchSuffix}_devel = $portVersion
 	devel:libmcrypt$secondaryArchSuffix = $libVersion
+	cmd:libmcrypt_config$secondaryArchSuffix = $portVersion
 	"
```

Reply:
@fbrosson Do we also need to add it at `PROVIDES`?

Reply:
No, it's only needed in `PROVIDES_devel` because the `libmcrypt-config` command will only be in the devel subpackage. (There are no files in bin/ shipped in the base package, for this recipe.)